Tests/Form/Type/DocumentTypeTest.php

```diff
@@ -30,7 +30,7 @@ public function setUp()
 
     public function testDocumentManagerOptionSetsEmOption()
     {
-        $field = $this->factory->createNamed('name', 'document', null, array(
+        $field = $this->factory->createNamed('name', 'Doctrine\Bundle\MongoDBBundle\Form\Type\DocumentType', null, array(
```

Comment:
If you support PHP 5.5+, you should use `DocumentType::class` and import the statement.